roboplot/core/plotter.py

```diff
@@ -0,0 +1,76 @@
+import roboplot.core.curves as curves
+import roboplot.core.debug_movement as debug_movement
+import roboplot.core.liftable_pen as liftable_pen
+import roboplot.core.stepper_control as stepper_control
+
+
+class Plotter:
+    default_pen_speed = 100  # I.e. as fast as possible
+    default_resolution = 0.1
+
+    def __init__(self, axes: stepper_control.AxisPair, pen: liftable_pen.LiftablePen):
+        self._axes = axes
+        self._pen = pen
+
+    def home(self):
+        self._pen.lift()
+        self._axes.home()
+
+    def draw(self, curve_list, pen_speed: float = default_pen_speed, resolution:float=default_resolution):
+        """Draw the curve (with the pen down), lifting the pen before and after."""
```

Comment:
I have to say, I was in two minds as to whether to bother commenting this method at all, since I feel I'm just writing the algorithm twice - once in the code and once in the comment. But I'll change it because we need to get this merged asap!!